src/app/leaderboard/page.tsx

```diff
@@ -0,0 +1,182 @@
+"use client";
+import Image from "next/image";
+import { Trophy, Award, Medal, Ribbon } from "lucide-react";
+import { useState } from "react";
+
+export default function Home() {
+  const [alltimemode, setmode] = useState(false);
+
+  return (
+    <li className="w-full overflow-hidden gap-7 flex flex-col items-center bg-[#161616]">
+      <p className="mt-5 text-center text-white text-[3rem] sm:text-[2.5rem] md:text-[4.2rem] lg:text-[5rem] font-medium">
+        Leaderboard
+      </p>
+
+      {/* Alltime/Monthly Switch */}
+      <div className="flex justify-evenly items-center rounded-xl bg-[#1C1C1C] w-[50%]">
+        <button
+          style={{ backgroundColor: alltimemode ? "#292929" : "#1C1C1C" }}
+          className=" transition-colors duration-300 flex w-[50%] justify-center items-center rounded-md"
+          onClick={() => setmode(!alltimemode)}
+        >
+          <p className="text-lg font-semibold text-white mt-2 ml-2 mb-2">
+            Month
+          </p>
+        </button>
+        <button
+          style={{ backgroundColor: !alltimemode ? "#292929" : "#1C1C1C" }}
```

Comment:
having hardcoded colors makes it hard to manage the colors on a later update, add the color to `tailwind.config.js` and use tailwind conditional classes 
eg:

```suggestion
         className={`transition-colors duration-300 flex w-[50%] justify-center items-center rounded-md ${
  alltimemode ? "bg-somecolor" : "bg-othercolor"
}`}

```
 
}`}